### ISSUE TITLE
hnix

### ISSUE THREAD

**Issue by jwiegley** [Gabriel439/nixfmt#6]
Hi Gabriel, hnix has both a full parser and a pretty-printer, which would make your work on this project much easier.

**Comment by Gabriella439** [comment]
Yeah, I basically consider this project dead and it made a lot of bad choices because it was a hackathon project.  I only open sourced it so people could browse all my mistakes :)

**Comment by jwiegley** [comment]
Do you no longer have a need/desire for a Nix formatter?

**Comment by dtzWill** [comment]
(I sure do!)

**Comment by Gabriella439** [comment]
Yeah, I don't use a Nix formatter for my development.  It was purely a project of curiosity
